Tools/Scripts/webkitpy/style/checker.py

```diff
@@ -343,6 +343,10 @@
      os.path.join('Source', 'WebKit', 'Shared', 'Extensions', 'WebExtensionSQLiteHelpers.h')],
      ["-readability/naming/underscores"]),
 
+    ([  # This file triggers false positives in the constructor initialization list (due to lambda usage), which looks like wrongly indented namespace code to the style checker
+     os.path.join('Source', 'WebKit', 'WebProcess', 'WebPage', 'CoordinatedGraphics', 'ThreadedCompositorPlayStation.cpp')],
+     ["-whitespace/indent"]),
```

Comment:
This is a common false positive, I would just ignore the style failure instead of ignoring all possible indent issues in that file.